queries/yaml/highlights.scm

```diff
@@ -0,0 +1,31 @@
+(block_mapping_pair key: (flow_node) @field)
+(flow_pair key: (flow_node) @field) ; This one doesn't seem to work, no idea what's wrong with it.
+(boolean_scalar) @boolean
+(null_scalar) @constant.builtin
+(double_quote_scalar) @string
+(single_quote_scalar) @string
+(double_quote_scalar (escape_sequence)) @string.escape
+(single_quote_scalar (escape_sequence)) @string.escape
+(integer_scalar) @number
+(float_scalar) @number
+(comment) @comment
+(anchor) @type
+(alias) @type
+(tag) @type
+(yaml_directive) @keyword
+(ERROR) @error
+[
+; "+" ; including this causes "invalid node type" errors?!?
```

Comment:
It's not in that list: https://github.com/ikatyang/tree-sitter-yaml/blob/master/grammar.js#L511-L530 Btw. what is + supposed to do in yaml?

Reply:
https://yaml.org/spec/1.2/spec.html#id2794534

> Keeping is specified by the “+” chomping indicator.

Reply:
but I hadn't realized that literal characters needed to correspond to things described in the grammar, so I suppose I should just take this out for now

Reply:
You can experiment with this PR: https://github.com/nvim-treesitter/playground/pull/13

Reply:
You can also watch tree-sitter-yaml to support "+" as soon as they support it.